etc/a.gwf

```diff
@@ -146,6 +158,9 @@ default_lang=fr
 # Authorization for wizards to send images (no: cannot; yes: can)
 can_send_image=yes
 
+# If set to yes, dump the content of bas image to the file bad-image (location??)
+dump_bad_images
```

Comment:
It is dumped to `bad-image` file (probably where gwd was launch). It is a debugging tool that should not depend on the base being used in a more generic parameter.

i.e. should not be in .gwf file

Reply:
ok